Cake.Recipe/Content/gitversion.cake

```diff
@@ -16,6 +16,29 @@ public class BuildVersion
             throw new ArgumentNullException("context");
         }
 
+        var cakeVersion = typeof(ICakeContext).Assembly.GetName().Version.ToString();
+
+        try
+        {
+            var rootPath = BuildParameters.RootDirectoryPath;
+            rootPath = context.GitFindRootFromPath(rootPath);
+        }
+        catch (LibGit2Sharp.RepositoryNotFoundException rnfe)
+        {
+            context.Warning("Unable to locate git repository, so GitVersion can't be executed, returning default version numbers...");
+
+            return new BuildVersion
+            {
+                Version = "0.1.0",
+                SemVersion = "0.1.0",
```

Comment:
We should probably have a pre-release tag here, as to not confuse it with an actual `0.1.0` release.